src/components/ComponentIndexPage/component-index.scss

```diff
@@ -148,3 +165,45 @@
   color: $text-01;
   margin-bottom: $spacing-06;
 }
+
+.component-index-not-found__link {
+  @include carbon--type-style('body-long-02');
+
+  margin-bottom: $spacing-06;
+  padding: 0 $spacing-02 0 $spacing-02;
+}
+
+//----------------------------------------------------------------------------
+// Component Index Filter
+//----------------------------------------------------------------------------
+.component-index-filter-container {
+  margin-top: 12.5rem;
+  padding-left: $spacing-07;
+}
+// Header
+.component-index-filter__header {
+  @include carbon--type-style('heading-01');
+  color: $text-01;
+}
+
+// Filter fieldset & label
+.component-index-filter__fieldset  {
+  padding-bottom: $spacing-05;
+  margin-top: $spacing-05;
+}
+.component-index-filter__label  {
+  border-top: 1px solid $ui-03;
+  padding-top: $spacing-05;
+  padding-bottom: $spacing-03;
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  letter-spacing: 0.16px;
+  color: #525252;
```

Comment:
Is this a specific color token? Or should we use the variable for this value from the IDL swatch?

Reply:
updated :)